### ISSUE TITLE
Telegram notifications not working

### ISSUE THREAD

**Issue by pax0707** [Tautulli/Tautulli#566]
Sending test message works, notifications not.

```
2016-02-22 07:49:50 INFO    Writing configuration to file
2016-02-22 07:49:48 INFO    PlexPy Notifiers :: Telegram notification sent.
2016-02-22 07:49:47 INFO    Writing configuration to file
2016-02-22 07:49:47 DEBUG   Sending test Telegram notification.
2016-02-22 07:46:45 INFO    Re-scheduled background task: Check for active sessions
2016-02-22 07:46:45 INFO    Re-scheduled background task: Check for Plex remote access
2016-02-22 07:46:45 INFO    Re-scheduled background task: Check for recently added items
2016-02-22 07:46:45 INFO    Writing configuration to file
2016-02-22 07:45:18 ERROR   Uncaught exception: Traceback (most recent call last):
File "D:\Plex\PlexPy\plexpy\logger.py", line 221, in new_run
old_run(*args, **kwargs)
File "C:\Python27\lib\threading.py", line 763, in run
self.__target(*self.__args, **self.__kwargs)
File "D:\Plex\PlexPy\plexpy\notification_handler.py", line 272, in notify_timeline
set_notify_state(session=stream_data,
NameError: global name 'stream_data' is not defined
2016-02-22 07:45:18 INFO    PlexPy Notifiers :: Email notification sent.
2016-02-22 07:45:16 DEBUG   PlexPy Monitor :: Library item 212766 has been added to Plex.
2016-02-22 07:41:17 INFO    PlexPy is up to date
2016-02-22 07:41:17 DEBUG   Latest version is 5e094e7597b058073082af632237cd5ee03a92f1
2016-02-22 07:41:17 DEBUG   Requesting URL via GET method: https://api.github.com/repos/drzoidberg33/plexpy/commits/dev
```

**Comment by JonnyWong16** [comment]
You received the email notification though?

**Comment by pax0707** [comment]
Yes.

**Comment by JonnyWong16** [comment]
Fixed in v1.3.9.
